framework/src/actions/Action.C

```diff
@@ -43,10 +44,14 @@ Action::validParams()
   params.addPrivateParam<std::string>("action_type");
   params.addPrivateParam<ActionWarehouse *>("awh", nullptr);
 
+  params.addParam<std::vector<std::string>>(
+      "control_tags",
+      "Adds user-defined labels for accessing object parameters via control logic.");
```

Comment:
Is this documented somewhere?

Reply:
In the parameter list. The `Controls/index.md` unfortunately does not mention this. I can add something there.

Reply:
Yes, I realize there is a parameter description :) After reading that description, I still don't know how to use it. I looked in `Controls/index.md` as well and couldn't find it, so that'd be great if you could document that.